index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <div class="nav-bar">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
```

Comment:
- Please change the `<div class = "nav-bar">` to a `<nav class = "nav-bar">`, as that section hold the navigation menu and it will make your html more semantically correct. 

- [Optional] Please for the ham burger menu, I see you used images for each line, I would advise you just use one image that shows the three-line ham burger menu, so as they become one unit, or you can specifically wrap the three images inside its own `<div>` tag. (like below)
```html
<nav class="nav-bar">
  <div class = "ham-menu">
      <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
      <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
      <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
  </div>
</nav>
```
If you find it confusing, reach out to me on slack `@kanu mike chibundu`